tests/test_core.py

```diff
@@ -1462,6 +1462,11 @@ def test_amplitude_to_db_complex():
     assert np.allclose(db1, db2)
 
 
+def test_amplitude_to_db_scalar():
+    assert librosa.amplitude_to_db(1) == 0
+    assert np.isclose(librosa.amplitude_to_db(2), 6.0206)
```

Comment:
It looks like this test is now triggering failures in the type checking.  Probably the type annotations need to be revised to allow for scalar inputs.

Reply:
Looking at this now with the type overloads - do we need to do this also for the other db/amplitude/power converters?

Reply:
Not in this PR

Reply:
Can you clarify why not?  It seems like you're proposing to expand the functionality of one function in a family of related functions, but not the others.  We should keep this all internally consistent, especially since some of these are inverses of each other.

Reply:
Sorry, I wasn't clear. I don't have the ability right now to contribute more than this small change. It's the contrary of what you say, I expanded the functionality of amplitude_to_db to match the others, so this change improves the API consistency. Then I added a test as you requested, and then I fixed the typings :)

Reply:
Ok, thanks for clarifying!

I understand the point about expanding the API to match what's supported by other functions.  My question is about expanding the type annotations of the other functions to make this behavior (scalar input support) explicit, since we would now have it for only one of the set.

Reply:
(one month later) .. sorry to drop the thread here, but I'm not sure where we stand on this.  I do think that we should implement this change consistently across all relevant functions in this PR.  Is that something you'd be interested in doing, or should I try to chip in contributions toward that as well?

Reply:
You are more than welcome to chip in! I'm unfortunately still unable to continue contributing on this.